frontend/src/services.js

```diff
@@ -623,6 +623,12 @@ CoreServices.factory('Pulse', ['$resource', '$cookies', function($resource, $coo
             params: { pulseId: '@pulseId' },
             headers: { 'X-CSRFToken': () => $cookies.csrftoken },
         },
+        test: {
+            url: '/api/pulse/test',
+            method: 'POST',
+            params: { pulseId: '@id' },
```

Comment:
i think we should have this be at `/api/pulse/:id/test` which has been our normal convention for endpoints that represent a specific object.